CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   roles.
 - Update `k8scloudconfig` version to `v10.0.0` to include change for Kubernetes 1.19.
 - Allow configuration of `MINIMUM_IP_TARGET` and `WARM_IP_TARGET` for AWS CNI via annotations on `AWSCluster`
+- Add support for tagging AWS resources, managed by the operator, based on the custom resource labels.
```

Comment:
10.0.0 is already released you need to add the  changelog into the `Unreleased` part

Reply:
You I right, I miss read it